Modules/posixmodule.c

```diff
@@ -5716,7 +5716,7 @@ os_nice_impl(PyObject *module, int increment)
 
     /* There are two flavours of 'nice': one that returns the new
        priority (as required by almost all standards out there) and the
-       Linux/FreeBSD one, which returns '0' on success and advices
+       Linux/FreeBSD one, which returns '0' on success and advises
```

Comment:
Please revert. `advices` is perfectly legal.

Reply:
Advices seems to be plural of the noun while advised is the verb. I think this is right.